key.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
`go generate` is not integrated into `go test` et al., so putting this into `go generate` would mean that users cannot run the tests until they create these certificates.